network_test.go

```diff
@@ -253,7 +254,7 @@ func testNetworkMachineCNI(t *testing.T, useConfFile bool) {
 	}
 	fctesting.RequiresRoot(t)
 
-	cniBinPath := []string{"/opt/cni/bin", testDataBin}
+	cniBinPath := []string{"/testdata/bin", testDataBin}
```

Comment:
```suggestion
	cniBinPath := []string{testDataBin, "/opt/cni/bin"}
```

`testDataBin` is defined here and should provide the path to these CNI plugins built for ARM: https://github.com/firecracker-microvm/firecracker-go-sdk/blob/4648d28a3cc3018cf576cc8ae3cf696cb4c0dbc4/machine_test.go#L71

I don't think we can guarantee "/testdata" exists.

Reply:
Previously CNI plugins are set towards `/opt/cni/bin` which consists of binaries in X86 format. But when we require the  ARM related binaries and they exists in `testdata/bin`.

Reply:
Yes, we should put the testdata binaries on PATH before "/opt/cni/bin"; `testDataBin` gives us a reference to the path to testdata binaries. "/testdata/bin" is not guranteed to exist on all setups so we should use the `testDataBin` variable https://github.com/firecracker-microvm/firecracker-go-sdk/blob/8378e6a6b128ad730215c58ea00c9da2f26df87a/machine_test.go#L69-L71

Reply:
Changed the variable to `testDataBin`